repository/m.json

```diff
@@ -1295,6 +1295,18 @@
 				}
 			]
 		},
+		{
+			"name": "Memorize",
+			"details": "https://github.com/ckm2k1/sublime_memorize",
+			"labels": ["call stack", "memory aid"],
+			"releases": [
+				{
+					"sublime_text": ">=4180",
+					"tags": true,
+					"platforms": ["osx", "linux", "windows"]
```

Comment:
Since you now support all known platforms, just remove that line 😉

Reply:
Done